docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+- `aws/kinesis-firehose`: `symopsio/kinesis-firehose-addon/aws@1.1` ([View in Terraform registry](https://registry.terraform.io/modules/symopsio/kinesis-firehose-addon/aws/latest))
+- `aws/kinesis-data-stream`: `symopsio/kinesis-data-stream-addon/aws@1.1` ([View in Terraform registry](https://registry.terraform.io/modules/symopsio/kinesis-data-stream-addon/aws/latest))
+
+### Refactoring the `aws/secretsmgr` Addon
+If `aws/secretsmgr` is declared as an addon, update your configuration to add the `symopsio/secretsmgr-addon/aws` module explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+  
+  addons = ["aws/secretsmgr"]
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+
+module "secrets_manager_access" {
+  source  = "symopsio/secretsmgr-addon/aws"
+  version = "~> 1.1"
+
+  environment   = var.environment
+  iam_role_name = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
+```terraform
+# The following blocks may be removed after applying the updated configuration
+moved {  
+  from = module.runtime_connector.module.aws_secretsmgr[0].aws_iam_policy.this
+  to   = module.secrets_manager_access.aws_iam_policy.this  
+}  
+  
+moved {  
+  from = module.runtime_connector.aws_iam_role_policy_attachment.aws_secretsmgr_attach[0]
+  to   = module.secrets_manager_access.aws_iam_role_policy_attachment.attach_secrets_manager_access[0]  
+}
+```
+
+### Refactoring the `aws/kinesis-firehose` Addon
+If `aws/kinesis-firehose` is declared as an addon, update your configuration to add the `symopsio/kinesis-firehose-addon/aws` module explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+  
+  addons = ["aws/kinesis-firehose"]
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+
+module "kinesis_firehose_access" {
+  source  = "symopsio/kinesis-firehose-addon/aws"
+  version = "~> 1.1"
+
+  environment = var.environment
+  iam_role_name = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
```

Comment:
```suggestion
Optionally, you may also use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
```